backend/dataall/modules/mlstudio/cdk/stacks.py

```diff
@@ -0,0 +1,98 @@
+""""
+Creates a CloudFormation stack for SageMaker Studio users using cdk
+"""
+import logging
+import os
+
+from aws_cdk import (
+    cloudformation_include as cfn_inc,
+    Stack,
+)
+
+from dataall.modules.mlstudio.db.models import SagemakerStudioUser
+from dataall.db.models import EnvironmentGroup
+
+from dataall.cdkproxy.stacks.manager import stack
+from dataall.db import Engine, get_engine
+from dataall.db.api import Environment as EnvironmentRepository
+from dataall.utils.cdk_nag_utils import CDKNagUtil
+from dataall.utils.runtime_stacks_tagging import TagsUtil
+
+logger = logging.getLogger(__name__)
+
+
+@stack(stack='sagemakerstudiouser')
```

Comment:
If we rename stack name we could potentially have some issues since it's stored in the database

Reply:
I am also going to revert the stack class change, to avoid issues with CloudFormation re-creation of resources